.codacy.yml

```diff
@@ -2,7 +2,5 @@
 #  are not available for the project. For example a lot of
 #  package name contains capital letter and such names are conventional.
 exclude_paths:
-  - "eo-runtime/src/main/java/EOorg/EOeolang/EOrust.java"
-  - "eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java"
-  - "eo-maven-plugin/src/it/hash_package_layer/src/main/java/EOorg/EOeolang/Heaps.java"
-  - "eo-maven-plugin/src/it/hash_package_layer/src/main/java/EOorg/EOeolang/package-info.java"
\ No newline at end of file
+  - "eo-runtime/src/main/java/EOorg/EOeolang/EOfloat$EOeq.java"
+  - "eo-runtime/src/main/java/EOorg/EOeolang/EOint$EOeq.java"
```

Comment:
@c71n93 I think we don't need these lines here anymore

Reply:
@maxonfjvipon done